iotdb-core/datanode/src/test/java/org/apache/iotdb/db/utils/EnvironmentUtils.java

```diff
@@ -337,7 +337,7 @@ private static void createAllDir() {
     }
     // create user and roles folder
     try {
-      AuthorizerManager.getInstance().reset();
+      BasicAuthorizer.getInstance().reset();
```

Comment:
we should delete this, because there is no user and role folders in datanode any more.